tests/test_feature_extraction_deit.py

```diff
@@ -75,36 +75,6 @@ def prepare_feat_extract_dict(self):
             "image_std": self.image_std,
         }
 
-    def prepare_inputs(self, equal_resolution=False, numpify=False, torchify=False):
```

Comment:
Happy to move it out of the class but think it should stay in this file

Reply:
I removed it to the general `test_feature_extreaction_common.py` (a bit like the `ids_tensor` and `random_attention_mask` in `test_modeling_common.py`).